test/lib/omniauth/strategies/openid_connect_test.rb

```diff
@@ -278,6 +278,65 @@ def test_callback_phase_with_discovery # rubocop:disable Metrics/AbcSize
         strategy.callback_phase
       end
 
+      def test_callback_phase_with_no_state_without_state_verification # rubocop:disable Metrics/AbcSize
+        code = SecureRandom.hex(16)
+        nonce = SecureRandom.hex(16)
```

Comment:
This line can be dropped now that `nonce` has been cleaned up in https://github.com/omniauth/omniauth_openid_connect/pull/132.